README.md

```diff
@@ -504,6 +504,17 @@ For more details, run:
 ./test/e2e/run.sh
 </pre>
 
+**Run only a singular test**
+Edit the `cypress.json` file located in the `test/e2e/` folder.
```

Comment:
This would be easier with using [`only`](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Excluding-and-Including-Tests) :)

Reply:
Maybe I can keep this here and just say - "if you want to run a subset of tests for debugging" and then add "only" on running a singular test?